core/include/mm/tee_mmu.h

```diff
@@ -35,34 +35,47 @@
 /*-----------------------------------------------------------------------------
  * Allocate context resources like ASID and MMU table information
  *---------------------------------------------------------------------------*/
-TEE_Result tee_mmu_init(struct user_ta_ctx *utc);
+TEE_Result vm_info_init(struct user_ta_ctx *utc);
 
 /*-----------------------------------------------------------------------------
- * tee_mmu_final - Release context resources like ASID
+ * Release context resources like ASID
  *---------------------------------------------------------------------------*/
-void tee_mmu_final(struct user_ta_ctx *utc);
+void vm_info_final(struct user_ta_ctx *utc);
+
+/*
+ * Returns range of user mapping (excluding eventual permanent kernel mapping).
+ */
+void vm_info_get_user_range(struct user_ta_ctx *utc, vaddr_t *vstart,
+			    vaddr_t *vend);
+
+/*
+ * Creates a memory map of a mobj.
+ * Desired virtual address can be specified in @va or 0 if the next available
```

Comment:
```
- * Desired virtual address can be specified in @va or 0 if the next available
- * can be choosen.
+ * Desired virtual address can be specified in @va otherwise @va must be
+ * inited to 0 if the next available address can be chosen.
```

Reply:
Thanks, I'll updpate